packages/dashboard/src/components/feature-graph/styles.st.css

```diff
@@ -0,0 +1,16 @@
+#graph_root {
```

Comment:
why id?

Reply:
Actually your right, I don't need this style at all anyway